xontrib/xonda.xsh

```diff
@@ -111,6 +112,7 @@ def _deactivate():
        $PATH = $_DEFAULT_CONDA_PATH[:]
        del $_DEFAULT_CONDA_PATH
        del $CONDA_DEFAULT_ENV
+    del $CONDA_PREFIX
```

Comment:
Can you pop this inside the `if` block here?  Or add a separate check for `'CONDA_PREFIX' in ${...}`?  Otherwise this will throw a KeyError if someone runs deactivate before activating an env